packages/flutter_tools/lib/src/build_runner/web_fs.dart

```diff
@@ -105,9 +107,12 @@ class WebFs {
   }
 
   /// Retrieve the [DebugConnection] for the current application.
+  ///
+  /// if [useExtension] is true, will attempt to connect via the extension
```

Comment:
nit: "if" => "If"

Reply:
Also, are you sue "useExtension" is a thing? I don't see any other occurrences of it in this file 🤔

Reply:
..right, I removed it... silly me